app/src/main/java/woowacourse/movie/view/movielist/MovieListActivity.kt

```diff
@@ -1,40 +1,50 @@
 package woowacourse.movie.view.movielist
 
 import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
-import android.widget.ListView
 import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import woowacourse.movie.R
-import woowacourse.movie.domain.data.MovieData
-import woowacourse.movie.domain.data.MovieDummyData
-import woowacourse.movie.domain.movieinfo.Movie
 import woowacourse.movie.view.moviedetail.MovieDetailActivity
-import woowacourse.movie.view.viewmodel.toUIModel
+import woowacourse.movie.view.viewmodel.DummyData
+import woowacourse.movie.view.viewmodel.MovieListData
+import woowacourse.movie.view.viewmodel.MovieListData.ADData
+import woowacourse.movie.view.viewmodel.MovieUIModel
 
 class MovieListActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_movie_list)
 
-        setUpMovieDatas(MovieDummyData)
+        setUpMovieDatas()
     }
 
-    private fun setMovieData(movieData: MovieData): List<Movie> = movieData.getAll()
+    private fun setMovieData(): List<MovieListData> = DummyData.getItems()
 
-    private fun setUpMovieDatas(movieData: MovieData) {
-        val movieListView = findViewById<ListView>(R.id.movie_listView)
-        val movieListViewAdapter = MovieListViewAdapter(
-            setMovieData(movieData),
-            object : MovieListViewAdapter.OnBookListener {
-                override fun onClick(movie: Movie) {
+    private fun setUpMovieDatas() {
+
+        val movieListView = findViewById<RecyclerView>(R.id.movie_recyclerView)
```

Comment:
```suggestion
        val movieRecyclerView = findViewById<RecyclerView>(R.id.movie_recyclerView)
```
ListView에서 RecyclerView로 변환하셨다면, 변수명도 변경해보면 어떨까요?